README.md

```diff
@@ -60,158 +40,40 @@ git submodule update --init --recursive
 ## Building
 
 ```bash
-# export PATH=${LLVM_INSTALL_DIR}/bin:$PATH
 mkdir build
 cd build
 
 cmake .. \
  -DCMAKE_CXX_COMPILER=clang++ \
  -DCMAKE_C_COMPILER=clang \
- -DCMAKE_INSTALL_PREFIX=<install location> 
+ -DCMAKE_INSTALL_PREFIX=<install location>
  # optional: -DCMAKE_BUILD_TYPE=<Debug(default), Release, RelWithDebInfo>
-
+ # optional: to provide a path to separately built SPIRV-LLVM translator, use -DLLVM_SPIRV_BINARY=/path
```

Comment:
I doubt the -DLLVM_SPIRV_BINARY will help hipcc or Clang to locate llvm-spirv. AFAIK, ultimately the variable is used to set LLVM_SPIRV cmake variable and I'm not seeing anyone referencing it (or the LLVM_SPIRV_BINARY).

Reply:
correct, this will need 2nd part done in the clang